precompile/backend.go

```diff
@@ -0,0 +1,33 @@
+// (c) 2022-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package precompile
+
+import (
+	"context"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/vms/platformvm/teleporter"
+	"github.com/ava-labs/subnet-evm/plugin/evm/warp"
+)
+
+var _ Backend = (*noopBackend)(nil)
+
+// Backend defines the interface for precompiles to interact with vm backends.
+type Backend interface {
+	warp.WarpBackend
```

Comment:
Can we update this to just `AddMessage` instead of embedding `warp.WarpBackend` since we don't need the full interface.

Reply:
updated